daemon/mgr/container_stats.go

```diff
@@ -60,11 +88,18 @@ func (mgr *ContainerManager) StreamStats(ctx context.Context, name string, confi
 				return err
 			}
 
-			// metrics may be nil if the container is not running,
-			// so just ignore it and try get the metrics next time.
 			if metrics != nil {
 				containerStat := toContainerStats(metrics.Timestamp, c, stats)
-
+				//FIXME(ZYecho): should throw error when not found sandbox?
+				networkStat, err := mgr.getNetworkStats(name)
```

Comment:
I think it is better to return error here. is there any specific case?

ping @rudyfly

Reply:
in pouch, if container has no endpoint, the sandbox will be nil.

Reply:
when user uses `--net=none` for container, the `sandbox` will be nil. right?

Reply:
Yep, I think it's nil in this case.

Reply:
could we add the case for this? thanks

Reply:
add in api test.